topics/materials-science/tutorials/muspinsim/tutorial.md

```diff
@@ -0,0 +1,932 @@
+>      - *"Y axis"*: `asymmetry`
+>      - *"Fields"*:
+>        - *"Fields 1"*:
+>          - *"Field (T)"*: `1.0/muon_gyr`
+>      - *"Average axes"*: `orientation`
+>      - *"Euler Convention"*: `ZYZ`
+>      - *"Times"*:
+>        - *"Time 1"*:
+>          - *"Time"*: `range(0, 1.0, 50)`
+>    - *"Fitting Parameters"*:
+>      - *"Fit experimental data with simulations"*: `Yes`
+>      - *"Method to use to fit the data"*: `Nelder-Mead`
+>      - *"Variable to fit to the experimental datas"*:
+>        - *"Variable to fit to the experimental data 1"*:
+>          - *"Name of the variable"*: `g`
+>          - *"Starting value"*: `0.1`
+>          - *"minimum bound"*: `0.0`
+>          - *"maximum bound*: `inf`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `out_file` (Output dataset)
+>
+{: .hands_on}
+
+
+In order to perform the fitting, we have replaced the numerical *"Dissipation"*
+with a variable, `g`. We then need to provide details of this variable in the
+*"Fitting Parameters"* section: it's starting value, minimum and maximum bound.
+Finally we need to provide the method to use. Here we use
+`Nelder-Mead`{% cite nelder_mead_1965 %}, but
+`lbfgs`{% cite nocedal_wright_2006 %} is also supported. 
+
+## MuSpinSim Simulate
+
+At this stage of the process we need to include our "experimental" data that we
+wish to fit to, which should have been imported into our history.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Simulate](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim/muspinsim/1.1.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Configuration file containing simulation parameters"*: `Output dataset 'out_file' from step 1` (Input dataset)
+>    - {% icon param-file %} *"Experiment data to fit (.dat) (Optional)"*: `experiment.dat` (Input dataset)
+> 2. Should generate the following output(s):
+>    - {% icon param-collection %} `Muspinsim output collection` (Output dataset)
+>    - {% icon param-file %} `muspinsim log for muspinsim input file fitting` (Output dataset)
+>    - {% icon param-file %} `fit report for muspinsim input file fitting` (Output dataset)
+>
+{: .hands_on}
+
+Note that in addition to the usual outputs, which details the number of
+iterations and error on the final, fitted simulation.
+
+## MuSpinSim Plot
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Plot](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim_plot/muspinsim_plot/3.5.1+galaxy0) %} with the following parameters:
+>    - *"Plot Title"*: `Fitting Example`
+>    - *"Label for x axis"*: `t`
+>    - *"Label for y axis"*: `Asymmetry`
+>    - *"Output File Type"*: `PNG`
+>    - *"Muspinsim File Series"*:
+>      - *"Muspinsim File Series 1"*:
+>        - {% icon param-file %} *"Muspinsim Experiment Data (.dat)"*: `Output dataset 'muspinsim_results' from step 3`
```

Comment:
Label out of date

Reply:
Can't find any further references to `out_file` or `from step`. I would say I think you found them all (which is a huge help and really grateful you had the time to go through in such detail) but then I left them all in the first place so me thinking there are no errors isn't worth a whole lot.